_query-dsl/geo-and-xy/geopolygon.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Geopolygon
+parent: Geographic and xy queries
+grand_parent: Query DSL
+nav_order: 30
+---
+
+# Geopolygon query
+
+A geopolygon query returns documents with geopoints that are within the specified polygon. A document with multiple geopoints matches the query if at least one geopoint matches the query.
+
+A polygon is specified by a list of vertices in coordinate form. Unlike specifying a polygon for a geoshape field, the polygon does not have to be closed (specifying the first and last points as the same is unnecessary). Though points do not have to follow either clockwise or counterclockwise order, it is recommended that you list them in either of these orders. This will ensure the correct polygon is captured.
```

Comment:
```suggestion
A polygon is specified by a list of vertices in coordinate form. Unlike specifying a polygon for a geoshape field, the polygon does not have to be closed (specifying the first and last points at the same is unnecessary). Though points do not have to follow either clockwise or counterclockwise order, it is recommended that you list them in either of these orders. This will ensure that the correct polygon is captured.
```